modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/clay/BlogsEntryVerticalCard.java

```diff
@@ -133,12 +138,17 @@ public String getSubtitle() {
 			PortalUtil.getHttpServletRequest(renderRequest),
 			System.currentTimeMillis() - modifiedDate.getTime(), true);
 
-		return LanguageUtil.format(
-			_resourceBundle, "x-ago-by-x",
-			new Object[] {
-				modifiedDateDescription,
-				HtmlUtil.escape(_blogsEntry.getUserName())
-			});
+		Format shortDateFormat = 
+				getSimpleDateFormat("MM/dd/yy 'at' h:mm a", LocaleUtil.getDefault(), TimeZoneUtil.getDefault());
+		String startTag = "<span title=\"" + shortDateFormat.format(modifiedDate) + "\">";
+		String endTag = "</span>";
+		String text = LanguageUtil.format(
+				_resourceBundle, "x-ago-by-x",
+				new Object[] {
+					modifiedDateDescription,
+					HtmlUtil.escape(_blogsEntry.getUserName())
+				});
+		return startTag + text + endTag;
```

Comment:
This is more of a question for me. I notice we concatenate strings:

###### util-taglib/src/com/liferay/taglib/ui/IconTag.java
```
StringBundler sb = new StringBundler(5);

sb.append("event.preventDefault();");
sb.append(onClick);
sb.append("submitForm(document.hrefFm, '");
sb.append(HtmlUtil.escapeJS(getUrl()));
sb.append("')");

onClick = sb.toString();
```
instead of:
```
startTag + text + endTag
```
What's the reason? Safer, faster?

Reply:
AFAIK, the reason is it's faster starting from a certain number of concatenations. I'm not sure how many but I think starting from 4 concats and up it's faster to use `StringBundler`.

Reply:
I think Source Formatter will complain about it.